order/views.py

```diff
@@ -0,0 +1,82 @@
+import json
+
+from django.views import View
+from django.http import JsonResponse
+from django.db import transaction
+
+from cart.models import Cart
+from .models import Order, OrderItem, OrderStatus
+
+from user.utils import login_decorator
+
+class OrderpageView(View):
```

Comment:
페이지가 아니라 데이터를 기준으로 만들어주세요! 변수명도 마찬가지입니다~

```suggestion
class OrderView(View):
```